verticaltabs.js

```diff
@@ -617,6 +636,40 @@ VerticalTabs.prototype = {
     }
   },
 
+  resizeTabs: function () {
```

Comment:
It feels a new function isn't actually needed. This code would actually read cleaner if resize just used lambdas or bind to setup delay calls to itself.